### ISSUE TITLE
Updating `Training MNIST using Kubeflow, S3, and Argo` documentation

### ISSUE THREAD

**Issue by julienstroheker** [kubeflow/examples#102]
We went through this documentation using `minio` as S3 storage and a Kubernetes Cluster hosted on Azure and we found some issues.

I am planning to open a PR to share our learnings from that.

To avoid duplicate sections, I was thinking to create a new md file for minio and refer to it in the `README.md` in the same folder as few tweaks are necessary to make it work. 

WDYT?

@jlewi @wbuchwalter @ritazh @sozercan

**Comment by elsonrodriguez** [comment]
I was working on debugging this with @ritazh, and with their help figured out some of the minio-related issues.

I took some of the feedback here and added it to a pr: https://github.com/kubeflow/examples/pull/108

The ideal is to keep it so that not too much modification is needed to run on any given s3-compatible store, so I'd prefer that we keep everything in one document. To that end I just added some inline hints for minio users on how to configure the variables.

If that works let me know, or we can expand a section or two where needed, or create a new doc if we think it won't work.
